ocrd/ocrd/cli/workspace.py

```diff
@@ -323,6 +320,10 @@ def workspace_cli_bulk_add(ctx, regex, mimetype, page_id, file_id, url, file_grp
                 if param_name == 'url':
                     url_is_src = True
                     continue
+                elif param_name == 'file_id':
+                    # XXX file_id might be unset so we derive it once the other
+                    # replacements have happened
```

Comment:
We forgot to do the same for `--mimetype`. It is currently auto-filled above based on `file_path.suffix`, which is wrong. It should be `srcpath.suffix` below.

```suggestion
                elif param_name in ['mimetype', 'file_id']:
                    # auto-filled below once the other
                    # replacements have happened
```